src/signals/incident-management/containers/IncidentDetail/components/MapDetail/index.js

```diff
@@ -6,33 +6,25 @@ import { locationType } from 'shared/types';
 
 import './style.scss';
 
-const MapDetail = ({
-  value, hideAttribution, hideZoomControls, useSmallMarker, zoom,
-}) => {
+const MapDetail = ({ value, smallMarker, zoom, hideAttribution, ...otherProps }) => {
   const location = value && value.geometrie && value.geometrie.coordinates;
   const latlng = location ? { latitude: location[1], longitude: location[0] } : null;
   return (
     <div className="map-detail">
-      {latlng
-        ? (
-          <Map
-            latlng={latlng}
-            hideAttribution={hideAttribution}
-            hideZoomControls={hideZoomControls}
-            useSmallMarker={useSmallMarker}
-            zoom={zoom}
-          />
-        ) : ''}
+      {latlng ? (
```

Comment:
If nothing should be rendered when `latlng` has no value or isn't defined, should the notation be

```jsx
 {latlng && (
  <Map latlng={latlng} smallMarker={smallMarker} hideAttribution={hideAttribution} zoom={zoom} {...otherProps} />
)}
```